lib/capistrano/slack_notify.rb

```diff
@@ -117,6 +118,11 @@ def self.extended(configuration)
             set(:start_time, Time.now)
           end
 
+          desc "Notify Slack that the rollback has completed."
+          task :rolled_back do
+            post_to_channel(:yellow, "#{deployer} has rolled back #{deploy_target}")
```

Comment:
Yellow feels like a "pending" or "in-progress" action. I think of red as "something bad happened!" and of green as "success, well done!" Given that, I think this message best fits the "success, well done!" message, so `:green` would be most appropriate. Something was (1) completed, and (2) successful.